### ISSUE TITLE
Create dummy client

### ISSUE THREAD

**Issue by snollygolly** [snollygolly/open-hearts#36]
As we're developing the socket.io parts, we need a way to test specific actions in the game without using the socket.io tester (which sucks) or waiting on the front end logic (which may take a while, or be optimized for playing the game vs. testing it).

We need to create a page with some simple client side JS that will perform common game actions and show us responses/results.

The goal with the dummy client is information gathering, and making testing quick.
- [x] Needs to connect to the web socket instance
- [x] Needs to send a join event with a name and game id (should be input by the user)
- [x] Needs to be able to perform a fetch and get the current game state (not needed yet)

**Comment by cynical89** [comment]
/console is the dummy client it connects to websockets, ,lets you start a new game, lets you join a game, fetches game data, and lets you get the game state. more functionality to be added as the game progresses
